maps/Colosseum.asm

```diff
@@ -4,25 +4,25 @@
 
 Colosseum_MapScripts:
 	def_scene_scripts
-	scene_script .InitializeColosseum, SCENE_COLOSSEUM_INITIALIZE
-	scene_script .DummyScene1,         SCENE_COLOSSEUM_NOOP
-	scene_script .DummyScene2 ; unused
+	scene_script ColosseumInitializeScene, SCENE_COLOSSEUM_INITIALIZE
+	scene_script ColosseumNoop1Scene,               SCENE_COLOSSEUM_NOOP
```

Comment:
Some other SCENE constants may need realigning.
```suggestion
	scene_script ColosseumNoop1Scene,      SCENE_COLOSSEUM_NOOP
```

Reply:
Ah yeah, may have missed a few. I'll go back through and check them all.